packages/util/src/verkle.ts

```diff
@@ -293,18 +319,71 @@ export function decodeVerkleLeafBasicData(encodedBasicData: Uint8Array): VerkleL
  *   code size, and balance to be encoded.
  * @returns {Uint8Array} - A compact, binary-encoded representation of the account header basic data.
  */
-export function encodeVerkleLeafBasicData(basicData: VerkleLeafBasicData): Uint8Array {
-  const encodedVersion = setLengthLeft(int32ToBytes(basicData.version), VERKLE_VERSION_BYTES_LENGTH)
+export function encodeVerkleLeafBasicData(account: Account): Uint8Array {
+  const encodedVersion = setLengthLeft(int32ToBytes(account.version), VERKLE_VERSION_BYTES_LENGTH)
   // Per EIP-6800, bytes 1-4 are reserved for future use
   const reservedBytes = new Uint8Array([0, 0, 0])
-  const encodedNonce = setLengthLeft(bigIntToBytes(basicData.nonce), VERKLE_NONCE_BYTES_LENGTH)
+  const encodedNonce = setLengthLeft(bigIntToBytes(account.nonce), VERKLE_NONCE_BYTES_LENGTH)
   const encodedCodeSize = setLengthLeft(
-    int32ToBytes(basicData.codeSize),
+    int32ToBytes(account.codeSize),
     VERKLE_CODE_SIZE_BYTES_LENGTH,
   )
-  const encodedBalance = setLengthLeft(
-    bigIntToBytes(basicData.balance),
-    VERKLE_BALANCE_BYTES_LENGTH,
-  )
+  const encodedBalance = setLengthLeft(bigIntToBytes(account.balance), VERKLE_BALANCE_BYTES_LENGTH)
   return concatBytes(encodedVersion, reservedBytes, encodedNonce, encodedCodeSize, encodedBalance)
 }
+
+export const generateChunkSuffixes = (numChunks: number) => {
+  const chunkSuffixes: number[] = new Array(numChunks)
+  for (let x = 0; x < numChunks; x++) {
+    if (x < VERKLE_CODE_OFFSET) {
+      // Suffixes for chunks 0 - 127 start at 128 and end with 255
+      chunkSuffixes[x] = x + VERKLE_CODE_OFFSET
+      continue
+    }
+    if (x < 384) {
+      // Suffixes for chunks 128 - 383 start at 0 and go to 255
+      chunkSuffixes[x] = x - VERKLE_CODE_OFFSET
+      continue
+    }
+    if (x < 640) {
+      // Suffixes for chunks 384 - 639 start at 0 and go to 255
+      chunkSuffixes[x] = x - VERKLE_CODE_OFFSET - VERKLE_NODE_WIDTH
+      continue
+    }
+    if (x > 639) {
+      // Suffixes for chunks 640 - 793 start at 0 and go to 153
+      chunkSuffixes[x] = x - VERKLE_CODE_OFFSET - 2 * VERKLE_NODE_WIDTH
+      continue
+    }
+  }
+  return chunkSuffixes
+}
+
+export const generateCodeStems = async (
+  numChunks: number,
+  address: Address,
+  verkleCrypto: VerkleCrypto,
+) => {
+  // The maximum number of chunks is 793 (maxCodeSize - 24576) / (bytes per chunk 31) + (round up - 1)
```

Comment:
Why `-24576`?

Reply:
That's the maximum allowed code size [see here](https://github.com/ethereumjs/ethereumjs-monorepo/blob/8b7d4b19abf1834d15759a559adeffc85c06a57c/packages/evm/src/params.ts#L132)